--- a/target/linux/mvebu/cortexa72/base-files/etc/board.d/02_network
+++ b/target/linux/mvebu/cortexa72/base-files/etc/board.d/02_network
@@ -26,8 +26,9 @@
 marvell,armada8040-db)
 	ucidef_set_interfaces_lan_wan "eth0 eth2 eth3" "eth1"
 	;;
-marvell,armada7040-db)
-	ucidef_set_interfaces_lan_wan "eth0 eth2" "eth1"
+marvell,armada7040-db*)
+	ucidef_set_interfaces_lan_wan "eth0" "eth0.1"
+	ucidef_set_interface "dtdlink" device "eth0.2" protocol "static"
 	;;
 marvell,armada8040-clearfog-gt-8k)
 	ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4 eth2" "eth0 eth1"
